### PR TITLE
refactor to use random image ids and create a common image list funct…

### PR DESCRIPTION
**Pull request by tawalton**
…ion for nova and glance

### CODE REVIEW

File: mimic/model/images.py [hunk 1]
```diff
@@ -194,10 +205,12 @@ class RackspaceCoreOSImage(Image):
...
     """
-    images = {"CoreOS (Beta)": {"minRam": 512, "minDisk": 20, "OS-EXT-IMG-SIZE:size": 217868501},
-              "CoreOS (Alpha)": {"minRam": 512, "minDisk": 20, "OS-EXT-IMG-SIZE:size": 239664114},
+    images = {"CoreOS (Beta)": {"minRam": 512, "minDisk": 20, "OS-EXT-IMG-SIZE:size": 217868501,
+                                "id": Image.image_id()},
+              "CoreOS (Alpha)": {"minRam": 512, "minDisk": 20, "OS-EXT-IMG-SIZE:size": 239664114,
+                                 "id": Image.image_id()},
               "CoreOS (Stable)": {"minRam": 512, "minDisk": 20, "OS-EXT-IMG-SIZE:size": 224241308,
-                                  "com.rackspace__1__ui_default_show": "True"}}
+                                  "com.rackspace__1__ui_default_show": "True", "id": str(uuid.uuid4())}}
```
Comment (tawalton) [inline mimic/model/images.py:214]: this should be Image.image_id() instead of str(uuid.uuid4())

File: mimic/model/glance_objects.py [hunk 2]
```diff
@@ -4,18 +4,7 @@
 
 from characteristic import attributes, Attribute
 from json import dumps
-import uuid
-from mimic.model.images import (RackspaceWindowsImage, RackspaceArchImage,
```
Comment (ghost) [inline mimic/model/glance_objects.py:8]: +1 to simplified imports, I like it

**Comment by ghost** [comment]
long as tests pass and it works in reach, this LGTM!

File: mimic/model/images.py [hunk 3]
```diff
@@ -19,6 +20,13 @@ def set_is_default(self):
         """
         self.is_default = True
 
+    @classmethod
+    def image_id(cls):
```
Comment (ghost) [inline mimic/model/images.py:24]: I like this being random since it closer models how the api actually behaves(id's frequently changing)

**Pull request merged**